packages/network-clients/codegen.yaml

```diff
@@ -0,0 +1,32 @@
+schema: 'https://api.subquery.network/sq/subquery/kepler-testnet-subql-project'
```

Comment:
make the schema link an env so that it can be switch when user connect with different network

Reply:
done